serving/src/main/java/feast/serving/service/BigQueryServingService.java

```diff
@@ -335,4 +213,14 @@ private String generateTemporaryTableName() {
     String suffix = guid.substring(0, Math.min(guid.length(), 10)).replaceAll("-", "");
     return String.format("temp_%s", suffix);
   }
+
+  private TableId generateUUIDs(Table loadedEntityTable) throws InterruptedException {
+    String uuidQuery =
```

Comment:
What are these UUIDs used for again? I am also seeing them being returned to the end user.

Reply:
They are used for ensuring that only a single value is joined to the left hand column.

Reply:
Can we drop them at the end instead of returning them?